xgi/classes/reportviews.py

```diff
@@ -391,14 +391,4 @@ def members(self, e=None, dtype=list):
         if e not in self:
             raise IDNotFound(f'ID "{e}" not in this view')
 
-        try:
-            return self._id_dict[e].copy()
-        except IDNotFound:
-            if e is None:
-                if dtype is dict:
-                    return {key: self._id_dict[key] for key in self._ids}
-                elif dtype is list:
-                    return [self._id_dict[key] for key in self._ids]
-                else:
-                    raise XGIError(f"Unrecognized dtype {dtype}")
-            raise IDNotFound(f"Item {e} not in this view")
```

Comment:
This is all repeated from above, no? This duplication looks like it might have been the product of a botched merge/rebase (of which I am notoriously guilty). Good catch.

Reply:
Thanks! Yeah, I couldn't see a case where it would ever get used, so I removed.